src/main/kotlin/org/objectionary/ddr/sources/Sources.kt

```diff
@@ -80,6 +82,9 @@ class SrsTransformed(
      *
      * @param filename path to current xmir file in source directory
      * @return path to the temporary directory with xmir files
+     *
+     * @todo #121:30min if output directory already exists createFile throws exception and logger prints error message. This situation should be handled correctly.
+     * @todo #121:30min rename "tmpPath" to "resPath" and make it a public field.
```

Comment:
@c71n93 Is it worth creating such an issue? Looks like a refactoring that can be done in this pull request

Reply:
@Graur There are more changes than I expected. I think, this issue can be resolved together with others issues for this class (there are 3 of them). When I was making these todos, I was going to solve them in the next one PR. What do you think?

Reply:
@c71n93 Sure, let's keep this `todo`